runtime/README.rst

```diff
@@ -70,7 +70,7 @@ The following table shows the available devices along with supported features:
      - All observables; Finite-shots not supported
    * - Variance
      - Only for ``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``, and ``Hadamard``; Finite-shots not supported
-     - Not supported
+     - All observables; Finite-shots not supported
```

Comment:
So lightning.kokkos has overtaken lightning.cpu in feature support (if cpu doesn't support variance on all observables)? :D

Reply:
Even, Lightning has now support for all observables if we build it against the master branch :D I'll update that support within the next release of Lightning ;)